TeacherCourse/models.py

```diff
@@ -0,0 +1,70 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+Category = [
+    ('BUSINESS_AND_MANAGEMENT', 'Business & Management'),
+    ('COMPUTER_SCIENCE_AND_IT', 'Computer Science & IT'),
+    ('CREATIVE_ARTS', 'Creative Arts'),
+    ('ECONOMICS', 'Economics'),
+    ('HISTORY', 'History'),
+    ('HEALTH_AND_PSYCHOLOGY', 'Health & Psychology'),
+    ('LITERATURE', 'Literature'),
+    ('LAW', 'Law'),
+    ('LANGUAGES_AND_CULTURES', 'Languages & Cultures'),
+    ('MATHS', 'Maths'),
+    ('MARKETING', 'Marketing'),
+    ('POLITICS_AND_SOCITY', 'Politics & Socity'),
+    ('SCIENCE_AND_ENGINEERING', 'Science & Engineering'),
+    ('SPORT_AND_LEISURE', 'Sport & Leisure'),
+]
+
+
+class TeacherCoursesManager(models.Manager):
+
+    def in_price_range(self, min, max):
+        return self.filter(price__lte=max, price__gte=min)
+
+    def in_category(self, cat):
+        return self.filter(category=cat)
+
+    def got_experience(self, exp):
+        return self.filter(experience__gte=exp)
+
+
+class TeacherCourses(models.Model):
+
+    course_id = models.BigAutoField(primary_key=True)
+    # teacher_id = models.ForeignKey()
+    course_name = models.CharField(max_length=30, blank=False)
+    description = models.TextField()
+    price = models.PositiveIntegerField(default=0, blank=False)
+    objects = TeacherCoursesManager()
+
+    class Experience(models.IntegerChoices):
```

Comment:
I remembered this field being the expected experience level for the students who take this course. Makes sense now.

Reply:
> I think that maybe we should be changing the name of the field from Experience to Years of Experience or possibly changing the string that represent the options to : one year for 1 , two years 2 , etc.

I think it's a good idea to change the fields as you suggested so it will be clear to the user entering the details.

Reply:
I agree with you